content/ideas/eventlog-profiling.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Extract heap profiles from the event log
+---
+
+The [event log](https://downloads.haskell.org/~ghc/latest/docs/html/users_guide/eventlog-formats.html#eventlog-encodings) is a general purpose mechanism for understanding what a
+Haskell program is doing. It is a binary stream of low-level information logging
+what the RTS is doing.
+
+Ben Gamari [recently extended](https://ghc.haskell.org/trac/ghc/ticket/11094) the format so that heap profiling events are also
+written to the log. This means that, in theory, all the information present in a
```

Comment:
IIRC the heap profile in the eventlog framework isn't complete. Last I checked at least retainer and biographical profiling was not supported yet. Could we add it to the list of goals?

Reply:
Good point. I'll add that.